FF1Lib/FF1Rom.cs

```diff
@@ -364,12 +365,12 @@ public void Randomize(Blob seed, Flags flags, Preferences preferences)
 				FiendShuffle(rng);
 			}
 
-			if (flags.FormationShuffleMode != FormationShuffleMode.None)
+			if (flags.FormationShuffleMode != FormationShuffleMode.None && !flags.RandomizeEnemizer && !flags.RandomizeFormationEnemizer)

```

Comment:
Please make a bool function on flags that returns the result of ```!flags.RandomizeEnemizer && !flags.RandomizeFormationEnemizer```

Something like
`bool EnemizerEnabled => flags.RandomizeEnemizer || flags.RandomizeFormationEnemizer;`

and use it in all the places in this file where you need it.